src/game/savegame.c

```diff
@@ -497,7 +503,7 @@ void Savegame_ScanSavedGames(void)
     REQUEST_INFO *req = &g_SavegameRequester;
 
     req->items = 0;
-    for (int i = 0; i < MAX_SAVE_SLOTS; i++) {
+    for (int i = 0; i < g_Config.maximum_save_slots; i++) {
```

Comment:
We don't need `MAX_SAVE_SLOTS` anymore do we?

Reply:
Nope. I checked, and you can remove the `#define MAX_SAVE_SLOTS 16` in `const.h`.